integration/k8/geopm-prometheus-k8.yml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: geopm-service-pod
+spec:
+  selector:
+    matchLabels:
+      name: geopm-service
+  template:
+    metadata:
+      labels:
+        name: geopm-service
+    spec:
+      hostPID: true
+      tolerations:
+      containers:
+      - name: geopm-service
+        image: cmcantal/geopm-prometheus:latest
```

Comment:
Should we do anything about the fact that these point to your specific user's dockerhub?